src/storage-client/src/controller.proto

```diff
@@ -19,7 +19,7 @@ message ProtoCollectionMetadata {
     string blob_uri = 1;
     string consensus_uri = 2;
     string data_shard = 3;
-    string remap_shard = 4;
+    optional string remap_shard = 4;
```

Comment:
I imagined you checked, but is it a valid migration to make a previously non-optional field optional?

Reply:
Yes, this is allowed because it only removes a non-nullable constraint which, after checking with surfaces, was permitted.